Suppress unnecessary administrative UI elements

phenaproxima opened the pull request:
Addresses #2.

balsama commented:
Nice. Two small requests:

* Can we assert that the manage display tab is removed from at least one other entity besides nodes?
* We should probably remove the "manage display" link from bundle list page drop-buttons too.

phenaproxima commented:
Done. I added assertions for all node types and media types, as well as user accounts. And I implemented the surprisingly simple hook_entity_operation_alter() to remove the "Manage display" link from the drop buttons.
